classes/Validate.php

```diff
@@ -374,7 +374,7 @@ public static function isCatalogName($name)
      */
     public static function isMessage($message)
     {
-        return !preg_match('/[<>{}]/i', $message);
+        return Validate::isCleanHtml($message, false);
```

Comment:
```suggestion
        return static::isCleanHtml($message, false);
```
or
```suggestion
        return self::isCleanHtml($message, false);
```

If I am not wrong, this is a little better?

Reply:
@jolelievre self or static? :D

Reply:
@PierreRambaud, what do you think Pierre? 😝

Reply:
always prefer static when you allow the extension

Reply:
STATIC IT IS